state/protocol/badger/state.go

```diff
@@ -514,6 +514,33 @@ func (state *State) updateEpochMetrics(snap protocol.Snapshot) error {
 		return fmt.Errorf("could not update committed epoch final view")
 	}
 
+	currentEpochFinalView, err := snap.Epochs().Current().FinalView()
+	if err != nil {
+		return fmt.Errorf("could not update current epoch final view")
```

Comment:
```suggestion
		return fmt.Errorf("could not update current epoch final view: %w", err)
```
Wrap the error to maintain the context from the underlying failure.